include/View.hpp

```diff
@@ -39,12 +39,12 @@ class View : public ViewListeners
   void close();
   void focus_view();
 
-  static View *desktop_view_at(Server *server, double lx, double ly,
+  static View *desktop_view_at(Server &server, double lx, double ly,
```

Comment:
```suggestion
  static View *desktop_view_at(double lx, double ly,
```
(update in .cpp too)

Reply:
This can't work while the function is static

Reply:
Why? You just need to make a getInstance on the server singleton right?